src/api/firebase.js

```diff
@@ -72,11 +76,41 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 		totalPurchases: 0,
 	});
 }
+/**
+ * Update an existing item in the user's list in Firestore.
+ * @param {string} listId The id of the list we're adding to.
+ * @param {Object} item Information about the new item.
+ **/
+export async function updateItem(listId, item) {
+	const {
```

Comment:
The deconstructed object was moved from the component level to the Firebase API instead.